docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+## 1. Choose and purchase a Tx module and a compatible receiver
+
+There are many supported choices of mLRS hardware, including off-the-shelf options and  many DIY designs.
+We cover here three popular but mutually incompatible commercially available off-the-shelf options.  We summarize some pros and cons below to help you choose. Don't mix Rx and Tx between these options.
+
+### MatekSys mLRS [mR900-30-TX](https://www.mateksys.com/?portfolio=mr900-30-tx) module kit with their [mR900-30](https://www.mateksys.com/?portfolio=mr900-30) receiver
+- These 900 MHz band modules were designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- This option should provide the longest range of the three discussed here.
+- The mR900-30-TX module kit includes an HC04 module for a convenient Bluetooth serial connection to your ground station software running on a computer or Android device.
+- These modules are especially easy to flash via DFU over their USB C connection.
+- These use the SX126x LoRa chip and support 19 Hz, 31 Hz and FSK(50 Hz) modes.
+- This option does not work with any ELRS receivers.
+- The Tx module kit does not include an antenna or a JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- The [mR900-22](https://www.mateksys.com/?portfolio=mr900-22) is a compatible lower power receiver which might be appropriate for smaller vehicles.  But it is not yet recommended for new users because it does not include USB for flashing and improvements to ArduPilot serial passthrough to support firmware update without disconnecting the receiver from the flight controller are not yet released.
+
+### MatekSys mLRS [mR24-30-TX](https://www.mateksys.com/?portfolio=mr24-30-tx) module kit with their [mr24-30](https://www.mateksys.com/?portfolio=mr24-30)) receiver or any [Selected ELRS 2.4 GHz receiver](ELRS_RECEIVERS.md#selected-elrs-receivers)
+- These 2.4 GHz band modules were also designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- The mR24-30-TX module also includes the HC04 module but you may notice some interference since Bluetooth also uses the 2.4 GHz band.
+- These modules are also especially easy to flash via DFU over their USB C connection.
+- 2.4 GHz is a wider band which might support more simultaneous users.
+- These modules support 19 Hz, 31 Hz, 50, and FLRC(111 Hz) modes.
+- The Tx module kit does not include an antenna or JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- If you don't need quite as much range, you can opt for one of the smaller and lighter [Selected ELRS 2.4 GHz receivers](ELRS_RECEIVERS.md#selected-elrs-receivers) such as the RadioMaster RP4TD.
+- Some AIO flight controller boards include a low power 2.4 GHz ELRS serial receiver which can also be used with this Tx module.
```

Comment:
should we make a comment that ELRS receivers need to be flashed with mLRS firmware ?